docs/js/files/modules_FormModel.js.html

```diff
@@ -161,6 +161,12 @@ <h1 class="file-heading">File: modules/FormModel.js</h1>
     mapping: undefined,
 
     /**
+     * @property models
+     * @type Object
+     **/
+    models: undefined,
```

Comment:
I feel like for now, this is just making it easier for people to define the models on class level when they are defining new form model that extends the FormModel. Not sure what's the use case when people will want to change the models after initialization?